easybuild/easyconfigs/r/ReFrame/ReFrame-4.2.0.eb

```
@@ -58,6 +58,8 @@ sanity_pip_check = True
 modextrapaths = {
     # bootstrap script installs required dependencies to 'external' subdirectory
     'PYTHONPATH': 'external',
+    # hpctestlib should be a part of the PYTHONPATH
+    'PYTHONPATH': 'hpctestlib',
```

```suggestion
    # bootstrap script installs required dependencies to 'external' subdirectory
    # hpctestlib should be a part of the PYTHONPATH
    'PYTHONPATH': ['external', 'hpctestlib'],
```

I think, and it needs checking, that the `'PYTHONPATH': 'hpctestlib` overwrites `'PYTHONPATH': 'external'` instead of also adding `hpctestlib` to `PYTHONPATH`. So the items in the `external` directory are not in the `PYTHONPATH`.